core/src/main/java/kafka/autobalancer/LoadRetriever.java

```diff
@@ -428,8 +428,8 @@ public void retrieve() {
                 createTopic();
                 this.mainExecutorService.schedule(this::retrieve, 1, TimeUnit.SECONDS);
                 return;
-            } catch (Exception e) {
-                logger.error("Consumer poll error", e);
+            } catch (Throwable throwable) {
```

Comment:
We rerun the task on `Exception` is because `Exception` is expected to be recoverable, however for non-recoverable cases (like most of the `Error`), schedule another task is unneccessary becasue the current JVM status is unpredictable, so maybe just add catch `Throwable` after `Exception`, and logging it properly